### PR TITLE
Wip api module completed

### PR DESCRIPTION
**Pull request by realwebdev**
code review

### CODE REVIEW

File: controllers/activity.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+func CreateStartActivity(db *gorm.DB) gin.HandlerFunc {
+	return func(c *gin.Context) {
+		var activity models.Activity
+		if err := c.BindJSON(&activity); err != nil {
+			c.JSON(http.StatusNotFound, gin.H{
+				"message": "error occured in databinding",
+				"error":   err.Error()})
+			return
+		}
+		if err := models.CreateStartActivity(activity, db); err != nil {
```
Comment (Bilaltanveer38) [inline controllers/activity.go:22]: should be a line break above this

File: controllers/activity.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"github.com/jinzhu/gorm"
+	"github.com/realwebdev/Bilal/clockify3/models"
+)
+
+func CreateStartActivity(db *gorm.DB) gin.HandlerFunc {
+	return func(c *gin.Context) {
+		var activity models.Activity
```
Comment (Bilaltanveer38) [inline controllers/activity.go:15]: use short declaration operator

File: controllers/activity.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+				"error":   err.Error()})
+			return
+		}
+		if err := models.CreateStartActivity(activity, db); err != nil {
+			c.JSON(http.StatusNotFound, gin.H{
+				"message": "error occured while creating activity",
+				"error":   err.Error()})
+			return
+		}
+		c.JSON(http.StatusOK, fmt.Sprintf("activity %v has been created!", activity.Activity_name))
```
Comment (Bilaltanveer38) [inline controllers/activity.go:28]: should be a line break above this, as this is the last returning statement of the function

File: controllers/activity.go [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"github.com/jinzhu/gorm"
+	"github.com/realwebdev/Bilal/clockify3/models"
+)
+
+func CreateStartActivity(db *gorm.DB) gin.HandlerFunc {
```
Comment (Bilaltanveer38) [inline controllers/activity.go:13]: This should be named as StartActivity

Comment (realwebdev) [reply controllers/activity.go:13]: resolved

File: controllers/activity.go (hunk 2 above)
Comment (realwebdev) [reply controllers/activity.go:15]: resolved

File: controllers/activity.go (hunk 1 above)
Comment (realwebdev) [reply controllers/activity.go:22]: resolved

**Pull request closed without merging**